packages/field/src/FormControlMixin.js

```diff
@@ -76,7 +76,19 @@ export const FormControlMixin = dedupeMixin(
       }
 
       get inputElement() {
-        return (this.$$slot && this.$$slot('input')) || this.querySelector('[slot=input]'); // eslint-disable-line
+        return this.__getDirectSlotChild('input');
+      }
+
+      get labelElement() {
```

Comment:
I know it's already merged, but....:

- Are we sure none of our consumers uses $$slot?
- Shouldn't these be underscored/protected (meant to be used by Subclassers)?
- I think in all other components you introduced `_my**Node**` instead of `my**Element**`. Maybe align on this as well?